docs/UserGuide.adoc

```diff
@@ -143,20 +143,22 @@ Returns any person having names `Betsy`, `Tim`, or `John`
 Format: `help`
 
 === Importing a list of orders : `import`
-
-You will be able to import orders via .csv files that are given to you by your company.
+==== Description
+You will be able to import orders via csv file that are given to you by your company.
 The CSV file will include all relevant and important details of the parcels; such as the customers’ names,
 transaction id of the parcels and the address of the customers, etc. +
 
-Format: `import FILE NAME`
+==== Format
+* `import FILE NAME`
 
 ****
-* Import the specific csv with the file name in data folder to the application
+* Import the specific csv with the file name in data folder to the application.
+* Only *one* csv file can be import at one time.
 * The FILE NAME should include the extension. For example: *orders.csv*.
 * The CSV file should store at data folder that is at the same directory as the JAR file.
```

Comment:
Perhaps could include a more specific constraint here stating file can only be in .csv format